fe/fe-core/src/main/java/com/starrocks/http/rest/transaction/TransactionWithoutChannelHandler.java

```diff
@@ -0,0 +1,265 @@
+        Validate.isTrue(txnOperationParams.getChannel().isNull(), "channel isn't null");
+        this.txnOperationParams = txnOperationParams;
+    }
+
+    @Override
+    public ResultWrapper handle(BaseRequest request, BaseResponse response) throws UserException {
+        TransactionOperation txnOperation = txnOperationParams.getTxnOperation();
+        String dbName = txnOperationParams.getDbName();
+        String tableName = txnOperationParams.getTableName();
+        String label = txnOperationParams.getLabel();
+        LoadJobSourceType sourceType = txnOperationParams.getSourceType();
+        Long timeoutMillis = txnOperationParams.getTimeoutMillis();
+        Body requestBody = txnOperationParams.getBody();
+        LOG.info("Handle transaction without channel info, label: {}, source: {}", label, sourceType);
+
+        Database db = Optional.ofNullable(GlobalStateMgr.getCurrentState().getDb(dbName))
+                .orElseThrow(() -> new UserException(String.format("Database[%s] does not exist.", dbName)));
+
+        TransactionResult result = null;
+        switch (txnOperation) {
+            case TXN_BEGIN:
+                Table table = Optional.ofNullable(db.getTable(tableName))
+                        .orElseThrow(() -> new UserException(
+                                String.format("Table[%s.%s] does not exist.", dbName, tableName)));
+                result = handleBeginTransaction(db, table, label, sourceType, timeoutMillis);
+                break;
+            case TXN_PREPARE:
+                result = handlePrepareTransaction(
+                        db, label,
+                        Optional.ofNullable(requestBody.getCommittedTablets()).orElse(new ArrayList<>(0)),
+                        Optional.ofNullable(requestBody.getFailedTablets()).orElse(new ArrayList<>(0))
+                );
+                break;
+            case TXN_COMMIT:
+                result = handleCommitTransaction(db, label, timeoutMillis);
+                break;
+            case TXN_ROLLBACK:
+                result = handleRollbackTransaction(
+                        db, label,
+                        Optional.ofNullable(requestBody.getFailedTablets()).orElse(new ArrayList<>(0))
+                );
+                break;
+            case TXN_LOAD:
+                break;
+            default:
+                throw new UserException("Unsupported operation: " + txnOperation);
+        }
+
+        return new ResultWrapper(result);
+    }
+
+    private TransactionResult handleBeginTransaction(Database db,
+                                                     Table table,
+                                                     String label,
+                                                     LoadJobSourceType sourceType,
+                                                     long timeoutMillis) throws UserException {
+        if (!LoadJobSourceType.BYPASS_WRITE.equals(sourceType)) {
```

Comment:
Should we consider distinguishing by the type of management transaction rather than the source type? 
The bypass write seems to be a transaction maintained by FE, while the original stream load transaction is mainly maintained by BE, so some transaction management operations need to be forwarded.

Reply:
Yeah, The transaction management of bypass writing scenario is maintained by FE node, and won't redirect to BE. Maybe it's necessary to add a `txnType` field in `TransactionState` for distinguishing in the future.  Howeaver, In order to reduce confusion for users, i think it is appropriate to use `LoadJobSourceType` currently.

Reply:
I think create a new bypass write handler may be more clearer